### ISSUE TITLE
Compilation error in EditEntryWidget.cpp - incomplete type QButtonGroup

### ISSUE THREAD

**Issue by jaycyb** [keepassxreboot/keepassxc#2048]
Compilation fails on OSX when building with `release-tool build -v 2.3.3 -p "autotype networking sshagent"` and yields these errors in **EditEntryWidget.cpp** : 

```bash
/Users/jaycyb/opt/keepassxc/src/gui/entry/EditEntryWidget.cpp:81:42: error: allocation of incomplete type 'QButtonGroup'
    , m_autoTypeDefaultSequenceGroup(new QButtonGroup(this))
                                         ^~~~~~~~~~~~
/Users/jaycyb/opt/keepassxc/src/gui/entry/EditEntryWidget.h:37:7: note: forward declaration of 'QButtonGroup'
class QButtonGroup;
      ^
/Users/jaycyb/opt/keepassxc/src/gui/entry/EditEntryWidget.cpp:82:41: error: allocation of incomplete type 'QButtonGroup'
    , m_autoTypeWindowSequenceGroup(new QButtonGroup(this))
                                        ^~~~~~~~~~~~
/Users/jaycyb/opt/keepassxc/src/gui/entry/EditEntryWidget.h:37:7: note: forward declaration of 'QButtonGroup'
class QButtonGroup;
      ^
/Users/jaycyb/opt/keepassxc/src/gui/entry/EditEntryWidget.cpp:174:35: error: member access into incomplete type 'QButtonGroup'
    m_autoTypeDefaultSequenceGroup->addButton(m_autoTypeUi->inheritSequenceButton);
                                  ^
/Users/jaycyb/opt/keepassxc/src/gui/entry/EditEntryWidget.h:37:7: note: forward declaration of 'QButtonGroup'
class QButtonGroup;
      ^
/Users/jaycyb/opt/keepassxc/src/gui/entry/EditEntryWidget.cpp:175:35: error: member access into incomplete type 'QButtonGroup'
    m_autoTypeDefaultSequenceGroup->addButton(m_autoTypeUi->customSequenceButton);
                                  ^
/Users/jaycyb/opt/keepassxc/src/gui/entry/EditEntryWidget.h:37:7: note: forward declaration of 'QButtonGroup'
class QButtonGroup;
      ^
[ 57%] Building CXX object src/CMakeFiles/keepassx_core.dir/keys/FileKey.cpp.o
4 errors generated.
make[2]: *** [src/CMakeFiles/keepassx_core.dir/gui/entry/EditEntryWidget.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [src/CMakeFiles/keepassx_core.dir/all] Error 2
make: *** [all] Error 2
```

## Expected Behavior
build successful

## Current Behavior
build fails

## Possible Solution
Not tested: reverting to a previous version of QT

## Steps to Reproduce (for bugs)
```bash
./release-tool build -v 2.3.3 -p "autotype networking sshagent"
```

## Context
I've successfully built previous versions with either release-tool or make directly. But today I'm getting this compilation error.

Note: i have run a `brew update && brew upgrade` beforehand

## Debug Info
Libraries:
- qt5: 5.11.0
- libgcrypt: 1.8.2

Operating system: macOS High Sierra 10.13.4
CPU architecture: Intel x86_64 Core i7

Enabled extensions:
- autotype 
- networking
- sshagent

**Comment by hifi** [comment]
This was fixed after 2.3.3 in 3bbc6ac0e6298d27bfe0c41999460cafda8edf18. You can apply that patch for 2.3.3 if you compile against Qt 5.11.

**Comment by jaycyb** [comment]
Thanks @hifi that works 👍
